server/socket/socket.ts

```diff
@@ -145,6 +146,22 @@ export default async function startSocketIOServer() {
       credentials: true,
     },
   });
+  const db = (global.db.conn as Mongoose).connection.db;
```

Comment:
Should we use the dbConnect function here?

`const db = (await dbConnect()).connection.db;`